gemini/evaluation/metrics_based_gemini_evaluation.ipynb

```diff
@@ -0,0 +1,646 @@
+{
```

Comment:
<p>Use the <code>GenerationConfig</code> class from vertexai.generative_models&nbsp; </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/482/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>